kuma/wiki/management/commands/render_document.py

```diff
@@ -2,100 +2,115 @@
 """
 Manually schedule the rendering of a document
 """
+from __future__ import division
+
 import datetime
 import logging
+from math import ceil
 from optparse import make_option
 
+from celery import chain
+
 from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
 
 from kuma.wiki.helpers import absolutify
 from kuma.wiki.models import Document, DocumentRenderingInProgress
-from kuma.wiki.tasks import render_document
+from kuma.wiki.tasks import email_render_document_progress, render_document
+
+
+log = logging.getLogger('kuma.wiki.management.commands.render_document')
 
 
 class Command(BaseCommand):
-    args = "<document_path document_path ...>"
-    help = "Render a wiki document"
+    args = '<document_path document_path ...>'
+    help = 'Render a wiki document'
     option_list = BaseCommand.option_list + (
-        make_option('--all', dest="all", default=False,
-                    action="store_true",
-                    help="Render ALL documents"),
-        make_option('--min-age', dest="min_age", default=600,
-                    help="Documents rendered less than this many seconds ago "
-                         "will be skipped"),
-        make_option('--baseurl', dest="baseurl",
+        make_option('--all', dest='all', default=False, action='store_true',
+                    help='Render ALL documents'),
+        make_option('--min-age', dest='min_age', default=600,
```

Comment:
When passing in an override value, will that be converted into int or does datetime.timedelta accept strings as well?